www/js/controllers.js

```diff
@@ -8,18 +8,20 @@ angular.module('emission.controllers', ['emission.splash.updatecheck',
                                         'emission.survey.launch',
                                         'emission.stats.clientstats',
                                         'emission.incident.posttrip.prompt',
-                                        'customURLScheme'])
+                                        'customURLScheme',
+                                        'emission.splash.ionicNotification'])
```

Comment:
And given that the push notification code is not tested, please remove it from this PR and send a new PR for it - I don't want it to be in this release.